src/common/config/index.ts

```diff
@@ -315,7 +322,7 @@ export default class Config extends EventEmitter {
      */
     regenerateCombinedConfigData = () => {
         // combine all config data in the correct order
-        this.combinedData = Object.assign({}, this.defaultConfigData, this.localConfigData, this.buildConfigData, this.registryConfigData);
+        this.combinedData = Object.assign({}, this.defaultConfigData, this.localConfigData, this.buildConfigData, this.registryConfigData, {useNativeWindow: this.useNativeWindow});
```

Comment:
Just a suggestion, and we don't have to do this here, but I wonder if it makes sense for us to define a type for more of these little items that should be passed around with the config. One thing that comes to mind was what I mentioned before, passing around the tab objects used by the main process that contain info on how the tabs are constructed wrt. products and the relative URLs.